js/builder_graph.js

```diff
@@ -502,16 +502,19 @@ class SpellSelectNode extends ComputeNode {
  */
 function getDefenseStats(stats) {
     let defenseStats = [];
-    let def_pct = skillPointsToPercentage(stats.get('def'));
-    let agi_pct = skillPointsToPercentage(stats.get('agi'));
+    let def_pct = skillPointsToPercentage(stats.get('def')) * 0.867;    // lol
+    let agi_pct = skillPointsToPercentage(stats.get('agi')) * 0.951;    // lol2
```

Comment:
uhhhhhh where did these constants come from

should either have them as constants in code or document how they got here

Reply:
yeah actually i need to fix this lol
they're used in 2 separate places 🥴 bad code